src/python/pants/rules/core/test_list_roots.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
I would name this `list_roots_test.py` so that an alphanumeric sort of the dir listing puts the file and its test side by side.